app/src/main/java/org/oppia/app/topic/info/TopicInfoFragmentPresenter.kt

```diff
@@ -82,12 +82,7 @@ class TopicInfoFragmentPresenter @Inject constructor(
       fragment,
       Observer<Topic> { topic ->
         topicInfoViewModel.topic.set(topic)
-        topicInfoViewModel.topicDescription.set(
-          htmlParser.parseOppiaHtml(
-            topic.description,
-            fragment.requireView().findViewById(R.id.topic_description_text_view)
-          )
-        )
+        topicInfoViewModel.topicDescription.set(topic.description)
```

Comment:
This looks good. 
Only line number 123 has a ktlint issue. `private fun logInfoFragmentEvent(topicId: String){` single space before `{`

Reply:
Done.